### PR TITLE
Allow setting of annotations on the Service Accounts

### PR DESCRIPTION
**Pull request by sidick**
Allow setting annotations on the created service accounts to allow easier use of IRSA.

### CODE REVIEW

File: helm/robusta/templates/runner-service-account.yaml [hunk 1]
```diff
@@ -263,6 +263,12 @@ kind: ServiceAccount
 metadata:
   name: {{ .Release.Name }}-runner-service-account
   namespace: {{ .Release.Namespace }}
+  {{- if .Values.runner.serviceAccount.annotations }}
```
Comment (arikalon1) [inline helm/robusta/templates/runner-service-account.yaml:266]: can you move the annotations to the `runnerServiceAccount` helm value? (see line 272 below)

**Review by arikalon1 — APPROVED** [review]
Thanks for the PR @sidick 
Left 1 small comment

**Comment by sidick** [comment]
> Thanks for the PR @sidick Left 1 small comment

Should be fixed now

**Review by arikalon1 — APPROVED** [review]
Thanks for the contribution @sidick 
Looks good

**Pull request merged**